dune-project

```diff
@@ -1,3 +1,43 @@
-(lang dune 1.8)
+(lang dune 2.9)
+
 (name yara)
-(using fmt 1.1)
+
+(license Apache-2.0)
+
+(source
+ (github elastic/ocaml-yara))
+
+(generate_opam_files true)
+
+(maintainers "Hezekiah Carty <hezekiah.carty@elastic.co>")
```

Comment:
Adding your name in both `maintainers` and `authors` would be a very welcome addition.